scout/server/blueprints/variants/templates/variants/sv-variant.html

```diff
@@ -1,5 +1,6 @@
 {% extends "layout_bs4.html" %}
 {% from "utils.html" import comments_panel, pedigree_panel %}
+{% from "variants/utils.html" import modal_causative %}
```

Comment:
:+1:

Reply:
It's wonderful when the variant types are compatible!

Reply:
> It's wonderful when the variant types are compatible!

Right? :)